src/components/MyObservations/MyObservationsContainer.js

```diff
@@ -188,6 +191,30 @@ const MyObservationsContainer = ( ): Node => {
   // this component again after returning from ObsEdit
   const onScroll = scrollEvent => setMyObsOffset( scrollEvent.nativeEvent.contentOffset.y );
 
+  if ( isDebugMode() && !currentUser && observations.length > 0 ) {
+    return (
+      <MyObservationsSimple
```

Comment:
@kueda curious if you considered making `MyObservationsSimple` into its own navigation route? I liked the simplicity of letting the navigator [figure out conditionally](https://github.com/inaturalist/iNaturalistReactNative/blob/f84d09648d6b01f727436689c9a02ced0758892d/src/navigation/StackNavigators/TabStackNavigator.js#L162) which screen to show for ObsDetails.

Reply:
I did not consider it, but now that I am considering it, I wouldn't do it that way. That approach in the navigator assumes that we *always* show a different version of ObsDetails in debug mode, but the requirement here was to only show a different version of MyObs if the user is signed out and has more than one obs. To do that, I actually want almost all of the functionality in MyObservationsContainer intact, i.e. I want identical controller logic but different view logic.